apps/skateboard/web/elm/GameBoard.elm

```diff
@@ -154,7 +143,7 @@ view address model =
         , button
             [ disabled (hideSubmit model.candidate), Events.onClick address Submit ]
             [ text "Submit" ]
-        , h2 [] [ text (List.foldl (\c a -> a ++ c.letter) "" model.candidate) ]
+        , h2 [ class "candidate" ] [ text (List.foldl (\c a -> a ++ c.letter) "" model.candidate) ]
```

Comment:
Do you think it's a good idea to make this a helper function like you have letterStyle?